### PR TITLE
Portfolio

### PR DESCRIPTION
**Pull request by sodabarauf**
this is capstone project of HTML and CSS, here I did these things:
-followed GitHub workflow.
-setup linters.
-added descriptive REABME.md file.
-added index.html and style.css file.
-added tags like section, div, img, button, input, textarea, form, p, h1, h3, i, and some other tags.
-added some styles.
-added @media query.

### CODE REVIEW

**Comment by faridafaqiri** [comment]
Hi! dear sodaba 
your project has great design I really liked it.
it's also responsive and looks good at small screen.

**Comment by sodabarauf** [comment]
> Hi! dear sodaba your project has great design I really liked it. it's also responsive and looks good at small screen.

thanks, from your feedback Farida Jan.

**Comment by sodabarauf** [comment]
@ahzamir,
hello sir, I hope your being well.
my partner reviewed and provided me feedback, please review my project and get me your feedback.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,95 @@
...
+    align-items: center;
+    justify-content: center;
+    flex-flow: column;
+    text-align: center;
+}
+
+
+```
+
+
```
Comment (ahzamir) [inline README.md]: - Remove this

File: index.html [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.2/css/all.min.css" integrity="sha512-SnH5WK+bZxgPHs44uWIX+LLJAJ9/2PkPKZ5QiAj6Ta86w+fsb2TkcmfRyVX3pBnMFcV7oQPJkl9QevSCWr3W6A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  </head>
+  <body>
+    <header>
+      <nav class="navbar"> 
+        <ul>
+          <li><a href="#home"><i class="fa-solid fa-house"></i> Home</a></li>
+          <li><a href="#about_me"><i class="fa-solid fa-user"></i> About me</a></li>
+          <li><a href="#portfolio"><i class="fa-solid fa-paperclip"></i> portfolio</a></li>
```
Comment (ahzamir) [inline index.html]: Change the text to `Projects`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+          <button><a href="https://github.com/sodabarauf/product-landing-page">source code</a></button><br>
+          <button><a href=" https://sodabarauf.github.io/product-landing-page/">live demo</a></button>
+        </div>
+        <div class="box">
+          <div>
+            <img src="./assets/tribute page.jpg" alt="">
+          </div><br><br>
+          <h3>tribute page</h3>
+          <button><a href="https://github.com/sodabarauf/tribute">source code</a></button><br>
+          <button><a href="">live demo</a></button>
```
Comment (ahzamir) [inline index.html]: - Add the demo link for this project

Screenshot 2024-06-08 145746

File: index.html [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+            <button><a href=" https://sodabarauf.github.io/technical-documentation-page/">live demo</a></button>
+        </div>
+        <div class="box">
+          <div>
+            <img src="./assets/capture of portfolio.jpg" alt="">
+          </div><br><br>
+            <h3>portfolio</h3>
+            <button><a href="https://github.com/sodabarauf/portfolio">source code</a></button><br>
+            <button><a href="https://sodabarauf.github.io/portfolio/" id="demolink">live demo</a></button>
+        </div>
```
Comment (ahzamir) [inline index.html]: - Remove this project and instead add your survey form project here

Screenshot 2024-06-08 145803

File: style.css
Comment (ahzamir) [inline style.css:1]: - Add a background color for your navbar header as when I scroll down it does not have a background color and it's not looking great.
Screenshot 2024-06-08 145729

**Review by ahzamir — CHANGES_REQUESTED** [review]
@sodabarauf 
image

- Well done you have done a great job implementing lots of requirements there are some small changes you need to add to your project inorder to get the final approval for your project.

- Check the comments below

**Comment by sodabarauf** [comment]
@ahzamir 
sir, thanks from provided feedback, I implemented all your feedback.
I hope this be acceptable.
thank you.

**Pull request merged**